tsl/test/sql/compression_update_delete.sql

```diff
@@ -1068,3 +1070,73 @@ UPDATE join_test1 t1 SET value = t1.value + 1 FROM join_test1 t2 WHERE t2.time =
 SELECT * FROM chunk_status;
 ROLLBACK;
 
+-- test for disabling DML decompression
+SHOW timescaledb.enable_dml_decompression;
+SET timescaledb.enable_dml_decompression = false;
+
+\set ON_ERROR_STOP 0
+-- should ERROR both UPDATE/DELETE statements because the DML decompression is disabled
+UPDATE sample_table SET c3 = NULL WHERE c4 = 5;
+DELETE FROM sample_table WHERE c4 = 5;
+\set ON_ERROR_STOP 1
+
+-- make sure reseting the GUC we will be able to UPDATE/DELETE compressed chunks
+RESET timescaledb.enable_dml_decompression;
+SHOW timescaledb.enable_dml_decompression;
+
+BEGIN;
+-- report 0 rows
+SELECT count(*) FROM sample_table WHERE c4 = 5 AND c3 IS NULL;
+UPDATE sample_table SET c3 = NULL WHERE c4 = 5;
+-- report 10k rows
+SELECT count(*) FROM sample_table WHERE c4 = 5 AND c3 IS NULL;
+ROLLBACK;
+
+BEGIN;
+-- report 10k rows
+SELECT count(*) FROM sample_table WHERE c4 = 5;
+DELETE FROM sample_table WHERE c4 = 5;
+-- report 0 rows
+SELECT count(*) FROM sample_table WHERE c4 = 5;
+ROLLBACK;
+
+-- create new uncompressed chunk
+INSERT INTO sample_table
+SELECT t, 1, 1, 1, 1
+FROM generate_series('2023-05-04 00:00:00-00'::timestamptz,
+            '2023-05-04 00:00:00-00'::timestamptz + INTERVAL '2 hours',
+            INTERVAL '1 hour') t;
+
+-- check chunk compression status
+SELECT chunk_name, is_compressed
+FROM timescaledb_information.chunks
+WHERE hypertable_name = 'sample_table'
+ORDER BY chunk_name;
+
+-- test for uncompressed and compressed chunks
```

Comment:
You can ignore this comment. I think this test case is repetitive. We have lot of update/delete tests when this guc is turned ON.

Reply:
But this tests is to make sure we'll block if we touch both compressed and uncompressed chunks during UPDATE/DELETE execution